R/card.R

```diff
@@ -90,6 +90,7 @@ card <- function(
 
   attribs <- args[nzchar(argnames)]
   children <- as_card_items(args[!nzchar(argnames)], wrapper = wrapper)
+  children <- card_image_add_classes(children)
```

Comment:
Let's investigate what this would mean for `navset_card_*()`

Reply:
Turns out this works perfectly with `navset_card_*()`.


https://github.com/user-attachments/assets/fe485a18-fa0e-4ffa-bd9b-a9c7fdbb2395